pkg/channel/filter/filter.go

```diff
@@ -64,22 +66,15 @@ func FilterSemver(ch string, tags []string) (target *string) {
 			if ch == channel.StableChannel {
 				continue
 			}
-			// Ensure that we are comparing the prerelease with the requested
-			// channel.
-			pre := v2.Pre[0].String()
-			if pre != ch {
-				continue
-			}
```

Comment:
We no longer require that alphas are compared with alphas and betas are compared with betas.